src/main/java/uk/gov/hmcts/divorce/common/model/Application.java

```diff
@@ -270,6 +270,18 @@
     @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
     private LocalDateTime dateSubmitted;
 
+    @CCD(
+        label = "Is the information provided on the case correct?",
+        access = {DefaultAccess.class}
```

Comment:
shouldn't the access be set to applicant2?

Reply:
im not sure how thats done tbh, it seems like all of the fields are set to DefaultAccess.class

Reply:
discussed, ticket for this.